CLAUDE.md

```
@@ -160,54 +189,83 @@ Environment variables (can be set via `.env` file):
 
 ### Charles Schwab Setup Process
 
-1. Create Charles Schwab brokerage account (Charles Schwab International if outside US)
+1. Create Charles Schwab brokerage account (Charles Schwab International if
+   outside US)
 2. Register developer account at https://developer.schwab.com/
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)